app/src/main/java/com/intervaltimer/google/services/TimerService.java

```diff
@@ -374,6 +375,8 @@ public void startWorkout(long workoutTime, long restTime, long startTime, int se
      * Pause the currently working timer
      */
     public void pauseTimer() {
+        if (isPaused) return;
```

Comment:
Is this consistent with the current coding style they are using?

Reply:
You're right, they're using {this style;}. Changed.